sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/CommonSasQueryParameters.java

```diff
@@ -0,0 +1,123 @@
+package com.azure.storage.common.implementation;
```

Comment:
Missing licensing header.

This class will need to be outside of an implementation package given that it is used in public API, starting with JDK9+ consumers of the SDK won't be able to interact with it directly.

Reply:
Just added the header